Do not use uname(1) to detect architecture.

ARCH_DEF is now set from variables.am

--- Makefile.custom.orig	2019-12-15 09:27:58.759145863 +0000
+++ Makefile.custom
@@ -2,23 +2,6 @@
 # Copyright (c) 2018 the libgoldilocks contributors.
 # Released under the MIT License.  See LICENSE.txt for license information.
 
-UNAME := $(shell uname)
-ARCH := $(shell uname -m)
-
-ARCH_DEF := arch_32
-
-ifeq ($(ARCH),x86_64)
-ARCH_DEF = arch_x86_64
-endif
-
-ifeq ($(ARCH),arch64)
-ARCH_DEF = arch_ref64
-endif
-
-ifeq ($(ARCH),arm)
-ARCH_DEF := arch_arm_32
-endif
-
 # Subdirectories for objects etc.
 # Many of them are mapped to build/obj right now, but could be split later.
 # The non-build/obj directories are the public interface.
